src/rpiplatesrecognition/libs/plate_acquisition/signs_extraction.py

```diff
@@ -1,21 +1,27 @@
 import numpy as np
 import cv2 as cv
 
+from .draw_projection import *
+from .config_file import *
 
+"""
+from draw_projection import *
+from config_file import *
+"""
 
-def adaptive_threshhold(img:np.ndarray):
+
+def adaptive_threshhold(img:np.ndarray, parameters):
     #img = cv.cvtColor(img, cv.COLOR_BGR2GRAY)
     #ret,img=cv.threshold(img,100,255,cv.THRESH_BINARY)
     #ret,img=cv.threshold(img,65,255,cv.THRESH_BINARY)
-    img=cv.adaptiveThreshold(img,255,cv.ADAPTIVE_THRESH_MEAN_C,cv.THRESH_BINARY,11,2)
+    img=cv.adaptiveThreshold(img,255,cv.ADAPTIVE_THRESH_MEAN_C,cv.THRESH_BINARY,parameters.adaptive_threshhold_size,parameters.adaptive_threshhold_C)

```

Comment:
does this work? I think whitespace is broken here

Reply:
> does this work? I think whitespace is broken here

Yes, I checked it one more time and it worked fine